build/config/linux/pkg_config.gni

```diff
@@ -100,12 +100,17 @@ template("pkg_config") {
     if (defined(invoker.extra_args)) {
       args += invoker.extra_args
     }
+    if (defined(invoker.supports_isystem)) {
```

Comment:
I don't see this being used at all in this PR, can we remove it?

Reply:
This is used in the RDK SB sources, it is needed to fix a build with the RDK BSP c++ compiler (i.e. g++), where it cannot find the c++ system libraries when using `-isystem`.

Reply:
Done.